sharepoint/sharepoint-ps/sharepoint-online/Copy-SPOPersonalSitePage.md

```diff
@@ -0,0 +1,186 @@
+
+## EXAMPLES
+
+### -----------------------EXAMPLE 1-----------------------------
+
+```powershell
+Copy-SPOPersonalSitePage -SourceSite 'https://prepspo-my.spgrid.com/personal/testuser_onmicrosoft_com' -DestinationSite 'https://prepspo.spgrid.com/sites/testsite' -PageName 'TestPage.aspx' -DeleteSourcePage $false -Confirm 
+```
+
+Example 1 demonstrates how a SharePoint Online administrator can copy the SharePoint page named `TestPage.aspx` from `testuser`'s SharePoint pages library in OneDrive to the `testsite` SharePoint site with confirmation. The source page will not be deleted.
+
+### -----------------------EXAMPLE 2-----------------------------
+
+```powershell
+Copy-SPOPersonalSitePage -SourceSite 'https://prepspo-my.spgrid.com/personal/testuser_onmicrosoft_com' -DestinationSite 'https://prepspo.spgrid.com/sites/testsite' -PageName 'TestPage.aspx' -DeleteSourcePage $true
+```
+
+Example 2 demonstrates how a SharePoint Online administrator can move the SharePoint page named `TestPage.aspx` from `testuser`'s SharePoint pages library in OneDrive to the `testsite` SharePoint site. The source page will be deleted after the copy operation.
+
+### -----------------------EXAMPLE 3-----------------------------
+
+```powershell
+Copy-SPOPersonalSitePage -SourceSite 'https://contoso.microsoft.com/sites/sourcesite' -DestinationSite 'https://contoso.microsoft.com/sites/testsite' -PageName 'TestPage.aspx' -DeleteSourcePage $true -Confirm
+```
+
+Example 3 demonstrates how a SharePoint Online administrator can copy the SharePoint page named `TestPage.aspx` from a SharePoint site name `sourcesite` to the `testsite` SharePoint site with confirmation. The source page will be deleted after the copy operation.
+
+## PARAMETERS
+
+### -SourceSite
+
+Specifies the URL of the source SharePoint site containing the SharePoint pages to copy.
+
+```yaml
+Type: SpoSitePipeBind
+Parameter Sets: (All)
+Aliases:
+Applicable: SharePoint Online
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DestinationSite
+
+Specifies the URL of the destination SharePoint site where the SharePoint pages will be copied to.
+
+```yaml
+Type: SpoSitePipeBind
+Parameter Sets: (All)
+Aliases:
+Applicable: SharePoint Online
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PageName
+
+Specifies the name of the SharePoint page to copy. This parameter is required when not using the `-AllPages` switch.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+Applicable: SharePoint Online
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DeleteSourcePage
+
+Indicates whether to delete the source SharePoint page(s) after copying. If set to $true, the operation will move the SharePoint page(s) instead of copying.
+
+```yaml
+Type: Boolean
```

Comment:
I commented in the cmdlet PR that this should be a switch param too, instead of bool